database/seeds/UserSeeder.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
Waarom heb je het gesprek geresolved als er niks mee is gedaan? Ik had gezegd dat dit bestand mag kan.. @RyanWennekes